plugins/net_plugin/net_plugin.cpp

```diff
@@ -789,6 +807,29 @@ namespace eosio {
             ( "_lport", local_endpoint_port );
          return mvo;
       }
+
+      // security group management
+      //
+   private:
+      std::optional<chain::account_name> participant_name_;
+      std::atomic<bool> participating_{false};
```

Comment:
default to true, the security_group changing logic and logic for TLS connections should handle setting it false when it needs to be

Reply:
Changed default value